spec/request/api/v1/conditions_controller_spec.rb

```diff
@@ -0,0 +1,138 @@
+require "rails_helper"
+
+describe Api::V1::ConditionsController, type: :request do
+  let(:json_body) { JSON.parse(response.body, symbolize_names: true) }
+  let(:form) { create :form }
+  let(:routing_page) { create :page, form:, routing_conditions: [condition] }
+  let(:goto_page) { create :page, form: }
+  let(:condition) { create :condition }
+  let(:condition_id) { condition.id }
+
+  describe "#index" do
+    context "when no conditions exist for a page" do
+      let(:routing_page) { create :page, form: }
+
+      it "returns 200 and an empty json array" do
+        get "/api/v1/forms/#{form.id}/pages/#{routing_page.id}/conditions", as: :json
```

Comment:
all GET/PUTS etc paths should be using rails url helpers. I think the pages request specs need to be updated. forms request specs have been updated to use `xxxx_path` but pages hasn't when these request specs were first written we hadn't migrated over to rails so we couldn't use the url helpers until after the migration.

Reply:
I've created this PR to fix the pages request specs https://github.com/alphagov/forms-api/pull/250